src/azul/service/hca_response_v5.py

```diff
@@ -247,18 +247,34 @@ def return_response(self):
     def make_bundles(self, entry):
         return [{"bundleUuid": b["uuid"], "bundleVersion": b["version"]} for b in entry["bundles"]]
 
+    def make_analysis_protocol(self, protocol):
```

Comment:
This is a bit fragmented. Do we need individual functions or can we inline them? I only see one call site.